### DISCUSSION TITLE
How can I apply for the GitHub Students Developer Pack

### DISCUSSION THREAD

**Discussion by HimanshuSharma0909** [community/community#125081] [GitHub Education] [answered]
Hello there,
I'm new here, can someone please tell me how can I apply for GitHub Students Developer Pack.

**Comment by nikhilsbuilds** [comment, marked as answer]
Sure, here are the steps to apply for the GitHub Student Developer Pack:

1. Go to the GitHub Education website (education.github.com).
2. Click on the "Get Benefits" button.
3. Sign in with your GitHub account or create a new one if you don't have one already.
4. Enter your school-issued email address and click "Request Authority to Verify."
5. GitHub will send a verification email to your school email address. Open the email and click the link to verify your student status.
6. Once your student status is verified, you'll be able to access the Student Developer Pack.

Let me know if you need any other assistance with the application process!

**Reply by Suryao07** [reply]
what if i don't have school-issued email

**Reply by nikhilsbuilds** [reply]
You can apply, but you need to attach some documents like an ID Card or something that mentions the date or something that confirms you are a student.

**Comment by HARI656-bytes** [comment]
i am still waiting for it when will i recive my student developer pack

**Comment by vanot6** [comment]
I have a student ID with my name on the front and the validity dates on the back. The application form doesn’t allow multiple file uploads, so every time I try to apply, my application gets rejected — either because the name or the date is ‘missing’. When I combine both sides into one file, the system says the photo has been ‘manipulated’. It’s a bit ridiculous. Is this intentional? Or is there any workaround?
